src/main/kotlin/blackjack/model/Card.kt

```diff
@@ -0,0 +1,48 @@
+package blackjack.model
+
+data class Card(val suit: Suit, val denomination: Denomination) {
+    override fun toString(): String {
+        return "${suit.name} ${Denomination.initial(denomination)}"
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        return "${suit.name} ${denomination.initial}"
```